python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/README.md

```diff
@@ -97,3 +118,18 @@ python ./alpaca_qlora_finetuning.py \
 {'loss': 1.8552, 'learning_rate': 2.9996503623845395e-05, 'epoch': 0.02}                                                                                                                           
   1%|█                                                                                                                                                         | 8/1164 [xx:xx<xx:xx:xx, xx s/it]
 ```
+
+### 4. Merge the adapter into the original model
+
+#### QLora
+
+```
+python ./export_merged_model.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH --adapter_path ./outputs/checkpoint-200 --output_path ./outputs/checkpoint-200-merged
+```
+
+#### QALora
```

Comment:
QA-LoRA

Reply:
Removed this section and use lora_config.json to determine the adapter type.